src/codegen/function_body_chunk.rs

```diff
@@ -182,9 +200,15 @@ impl Builder {
                      type_: None,
                  });
         body.extend(output_vars);
+
+        let mut ret_name = "_";
+        if trampoline.use_ret {
+            ret_name = "ret";
+        }
```

Comment:
Why not `let ret_name = if trampoline.use_ret { "ret" } else { "_" };`

Reply:
ah ! googled `let if else` but could only find `if let` documentation, tnx! (if you can't tell I'm not exactly fluent :)